tests/unit/test_api.py

```diff
@@ -79,3 +82,32 @@ def test_should_disallow_object(self):
             dehydrated_value(object())
         with self.assertRaises(TypeError):
             dehydrated_value(uuid4())
+
+
+def test_bookmark_class():
```

Comment:
I would recommend breaking these up into separate tests. Much easier to see what went wrong.